contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,227 @@
+from typing import List, Optional, Union
+
+from great_expectations.compatibility.typing_extensions import override
+from great_expectations.core import (
+    ExpectationConfiguration,
+)
+from great_expectations.execution_engine import ExecutionEngine
+from great_expectations.expectations.expectation import (
+    ExpectationValidationResult,
+    QueryExpectation,
+)
+from great_expectations.render import RenderedStringTemplateContent
+from great_expectations.render.renderer.renderer import renderer
+
+
+class ExpectColumnValuesToBePresentInAnotherTable(QueryExpectation):
+    """Expect the values in a column to be present in another table.
+
+     This is an Expectation that allows for the validation of referential integrity, that a foreign key exists in another table.
+
+     In the following example, order table has a foreign key to customer table, and referential integrity is preserved, because all of
+     the values of CUSTOMER_ID in order_table_1 are present in the CUSTOMER_ID column of customer_table.
+
+    "order_table_1": {
+         "ORDER_ID": ["aaa", "bbb", "ccc"],
+         "CUSTOMER_ID": [1, 1, 3],
+     }
+     "customer_table": {
+         "CUSTOMER_ID": [1, 2, 3],
+
+     }
+
+     However, in the second example, referential integrity is not preserved, because there are two values (4 and 5) in the CUSTOMER_ID
+     column of order_table_2 that are not present in the CUSTOMER_ID column of customer_table.
+
+     "order_table_2": {
+         "ORDER_ID": ["ddd", "eee", "fff"],
+         "CUSTOMER_ID": [1, 4, 5],
+     }
+     "customer_table": {
+         "CUSTOMER_ID": [1, 2, 3],
+
+     }
+     ExpectColumnValuesToBePresentInAnotherTable will PASS for example 1 and FAIL for example 2.
+
+     Args:
+         template_dict: dict containing the following keys:
+             foreign_key_column: foreign key-column of current table that we want to validate.
+             foreign_table: foreign table.
+             primary_key_column_in_foreign_table: key column for primary key in foreign table.
```

Comment:
Love this docstring, but took a second to map from the example to the params. Would it make sense to add an example of the `template_dict` for each of the two examples?

Reply:
The `template_dict` would be the same for the two examples, since the Expectation itself would be run on either `order_table_1` or `order_table_2` (our DataAssets).  I do agree that an example would be very helpful in reducing cognitive load. l add an example to the end of the docstring.